[WIP] Continued styles

Leland opened the pull request:
Continuing from #19, this adds a simple darker background to the main header and separates the individual pages' components into floating divs.

I've also increased the size of the Queueing button on the dispatch, and moved it into the header.

<img width="1221" alt="screen shot 2017-10-15 at 10 53 21 pm" src="https://user-images.githubusercontent.com/714017/31593279-b0c36f26-b1fb-11e7-98ed-6f166d7407d9.png">

<img width="1237" alt="screen shot 2017-10-16 at 8 11 28 am" src="https://user-images.githubusercontent.com/714017/31611344-b2ef0e06-b249-11e7-86c2-b2323890dfbf.png">
<img width="1222" alt="screen shot 2017-10-16 at 8 11 38 am" src="https://user-images.githubusercontent.com/714017/31611345-b3038aca-b249-11e7-8131-a90564535577.png">
<img width="1221" alt="screen shot 2017-10-16 at 8 12 35 am" src="https://user-images.githubusercontent.com/714017/31611371-cdccd17c-b249-11e7-8865-50ae5b2dc322.png">

nadavspi commented:
Queue is whack when you expand it. Might be nice to use the SD turquoise: #9ad7db.

Leland commented:
> Queue is whack when you expand it

(featurenotbug)

Leland commented:
Added a darker background, changed up a few minor CSS notes and fixed the queue whackness.

<img width="367" alt="screen shot 2017-10-17 at 10 38 46 pm" src="https://user-images.githubusercontent.com/714017/31698567-21a5fbfa-b38c-11e7-819f-24c140764dcb.png">
<img width="967" alt="screen shot 2017-10-17 at 10 39 41 pm" src="https://user-images.githubusercontent.com/714017/31698568-21b4fd12-b38c-11e7-8b87-33cc613bd065.png">

Leland commented:
Note to self: move dispatch items' remove link next to title rather than hour input
